app/models/manageiq/providers/cisco_intersight/physical_infra_manager/physical_switch_network_port.rb

```diff
@@ -1,5 +1,5 @@
 module ManageIQ::Providers::CiscoIntersight
-  class PhysicalInfraManager::PhysicalSwitchNetworkPort < ::PhysicalSwitchNetworkPort
+  class PhysicalInfraManager::PhysicalSwitchNetworkPort < ::PhysicalNetworkPort
```

Comment:
:+1: this is correct, I'm not sure this needs to be `PhysicalSwitchNetworkPort` and not `PhysicalNetworkPort` but can handle that separately.

Reply:
Honestly, I thought it was autocomplete that caused this and the next one. I did stare at the error a few times.  🤣

Reply:
Yeah they intended to call it this, here is the inventory collection showing the `physical_switch_network_ports` name and the `PhysicalNetworkPort` base class https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/inventory/persister/builder/physical_infra_manager.rb#L86-L92